### ISSUE TITLE
Timeout also displays traceback

### ISSUE THREAD

**Issue by hatlord** [piotrmurach/tty-command#48]
Hi,

I noticed that in more recent versions of TTY-Command (0.8.0+), a timeout will output a traceback to the terminal. I have already rescued and display an error message of my own, what is the best way of suppressing the traceback? 
Example included below.
```
#<Thread:0x00007ffca2a31820@/usr/local/lib/ruby/gems/2.5.0/gems/tty-command-0.8.1/lib/tty/command/process_runner.rb:159 run> terminated with exception (report_on_exception is true):
Traceback (most recent call last):
/usr/local/lib/ruby/gems/2.5.0/gems/tty-command-0.8.1/lib/tty/command/process_runner.rb:165:in `block in read_stream': TTY::Command::TimeoutExceeded (TTY::Command::TimeoutExceeded)
```

Thanks,

**Comment by piotrmurach** [comment]
Hi,

This traceback is unrelated to the `tty-command` version but Ruby version. I've checked that this starts to be the case from Ruby `2.5.0`. Charles Nutter has [submitted an issue](https://bugs.ruby-lang.org/issues/6647) to change Ruby behaviour to report errors in console when Thread terminates unexpectedly. In the past you needed to specify this behaviour explicility. I can see why this is a good thing since it provides a better transparency and prevents errors being swallowed by dying thread. However, I'm not sure what is the best solution in this case.

**Comment by piotrmurach** [comment]
Released `v0.8.2` with the fix.

**Comment by hatlord** [comment]
Worked perfectly for me, many thanks!
